indico/modules/receipts/controllers/templates.py

```diff
@@ -76,6 +76,9 @@ def _check_access(self):
         # Check that template belongs to this event or a category that is a parent
         if self.template.owner == self.target:
             return
+        # If the user can manage the template owner, no need to check further
+        if self.template.owner.can_manage(session.user):
```

Comment:
I don't think that's correct. You can preview a template inherited from a parent category, even without being a manager of that parent category.

Reply:
Hmm.. I'm having some trouble coming up with the logic for that.. I think we'd need to check if the owner is in the category chain in any category that the user can manage, right? Is there even a way to get that without checking the `can_manage` of every category in the instance?

I think that this simplification is fine for now tbh...

Reply:
Looking at the whole code I think your logic is fine. The case I had in mind (template inherited from parent category) is covered below with the chain check.